lighthouse-cli/bin.ts

```diff
@@ -62,7 +62,8 @@ const cliFlags = yargs
     'list-trace-categories',
     'config-path',
     'perf',
-    'port'
+    'port',
+    'timeout'
```

Comment:
I think we should be more specific. `--timeout` feels too generic and it's clear from this discussion, that it's ambiguous what it _could_ mean :) `page-load-timeout` or `max-wait-for-load` would be cool.